File: src/main/java/no/stunor/origo/eventorapi/model/origo/person/CompetitorPerson.java
package no.stunor.origo.eventorapi.model.origo.person;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;


@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class CompetitorPerson {
    String eventorId;
    String personid;
    PersonName name;
    Integer birthYear;
    String nationality;
    String gender;
}

File: src/main/java/no/stunor/origo/eventorapi/model/origo/CompetitorPerson.kt
package no.stunor.origo.eventorapi.model.origo

import no.stunor.origo.eventorapi.model.person.Gender
import no.stunor.origo.eventorapi.model.person.PersonName

data class CompetitorPerson (
        var eventorId: String = "",
        var personId: String?,
        var name: PersonName = PersonName(),
        var birthYear: Int = 0,
        var nationality: String = "",
        var gender: Gender = Gender.OTHER
)